FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/WebAuthn/WebAuthnAuthenticationTests.swift

```diff
@@ -75,7 +75,7 @@ class WebAuthnAuthenticationTests: WebAuthnSharedUtils {
             
             //  Perform registration
             let ex = self.expectation(description: "WebAuthn Registration")
-            callback.register(onSuccess: { (webAuthnOutcome) in
+            callback.register(usePasskeysIfAvailable: false, onSuccess: { (webAuthnOutcome) in
```

Comment:
Isn't it false by default?

Reply:
Yes it is, I added it on purpose as an example of how this flag can be overriden. I find it helpful for default values like that when searching the code to see an example use